eng/common-tests/matrix-generator/pr-matrix-samples/pr-matrix-generation-acceptance.tests.ps1

```diff
@@ -0,0 +1,41 @@
+Import-Module Pester
+
+# Load scenarios before we enter the Describe block so they're available for -ForEach
+$netScenarios = Get-Content (Join-Path $PSScriptRoot net_scenarios.json) | ConvertFrom-Json
+$pythonScenarios = Get-Content (Join-Path $PSScriptRoot python_scenarios.json) | ConvertFrom-Json
+
+Describe "Acceptance tests for .NET PR Matrix Generation" {
+    BeforeAll {
+        . $PSScriptRoot/pr-matrix-generation-acceptance.helpers.ps1
+        $RepoRoot = Get-Repo -Repo "Azure/azure-sdk-for-net" -Reference "331c07a1ab59ed0042972ca6d0df830df235280f"
```

Comment:
There may be an argument for just simulating the necessary files within common-tests, but I don't think it's strong enough to avoid just doing an old ref checkout. We may want to periodically keep this updated when we remember to do it.